src/extension.ts

```diff
@@ -198,21 +198,25 @@ exports.deactivate = deactivate;  // eslint-disable-line no-undef
 async function init () {
 	const isEnabled = await ExtensionContainer.context.globalState.get<boolean>(GlobalState.Enabled);
 	if (isEnabled) {
-		vscode.window.withProgress({ location: vscode.ProgressLocation.Window, title: 'Reading Appcelerator environment ...' }, async progress => {
-			if (await ExtensionContainer.context.globalState.get('titanium:liveview')) {
-				await vscode.commands.executeCommand('setContext', 'titanium:liveview', true);
-			}
-			appc.getInfo(async (error, info) => {
-				if (error) {
-					vscode.window.showErrorMessage('Error fetching Appcelerator environment');
-					return;
-				}
-				if (project.isTitaniumApp) {
-					await generateCompletions({ progress });
+		vscode.window.withProgress({ location: vscode.ProgressLocation.Notification, title: 'Reading Appcelerator environment ...' }, async progress => {
+			return new Promise((resolve, reject) => {
+				if ( ExtensionContainer.context.globalState.get('titanium:liveview')) {
+					vscode.commands.executeCommand('setContext', 'titanium:liveview', true);
 				}
-				// Call refresh incase the Titanium Explorer activity pane became active before info
-				await vscode.commands.executeCommand(Commands.RefreshExplorer);
-
+				appc.getInfo(error => {
+					if (error) {
+						vscode.window.showErrorMessage('Error fetching Appcelerator environment');
+						reject();
+					}
+					if (project.isTitaniumApp) {
+						generateCompletions({ progress });
+						resolve();
+					} else {
```

Comment:
I think this else needs to be removed but the code in the block kept

We kick off a refresh of the explorer because if a user has gone into the activity panel and viewed one of the tree items that gets populated via the appc info results (iOS->Simulator for example) prior to us loading the info it wont display, by kicking off the refresh we force the view to update.